Documentation/admin/logging-destination.md

```diff
@@ -127,8 +127,117 @@ Installing the x-pack plugin on your Elasticsearch nodes enables authentication
 
 ### AWS Elasticsearch
 
-AWS Elasticsearch isn't officially supported at this time. AWS Elasticsearch functions differently from standard Elasticsearch making the standard Fluentd Elasticsearch plugin not work with it by default. If you are interested in using or contributing support for AWS Elasticsearch, please file an issue on [Github](https://github.com/coreos-inc/tectonic-installer/issues). Alternatively, you could look at one of the many AWS Signing proxies, which may work for your purposes, and allow you to use the default Elasticsearch configuration, pointed at the signing proxy.
+AWS Elasticsearch Service to quickly and easily run logging aggregation infrastructure with minimal overhead. Let's walk through the steps we're going to take to do this is to setup our Elasticsearch Service on AWS.
+  		  
+ - Log on to the AWS console and click Elasticsearch service.
+ - Click "Create a new domain".
+ - Configure your instance type, disk, and permissions. We'll be using IAM based credentials for security. If you're not familiar with IAM users, you can read more about it [here](http://docs.aws.amazon.com/IAM/latest/UserGuide/id_users_create.html). Here's an example config that I made specifically for this purpose. 
```

Comment:
Configure the instance type, disk, and permissions. Use [IAM based credentials for security][iam-credentials].

Reply:
Done!